modules/dreamview/backend/simulation_world/BUILD

```diff
@@ -38,7 +38,7 @@ cc_library(
 
 cc_test(
     name = "simulation_world_service_test",
-    size = "small",
+    size = "large",
```

Comment:
Could you please note the reason for the change here?

Reply:
modify hdmap_common.cc, those tests become timeout, maybe those used  cache before.